Examples/LambdaFunctions/Package.swift

```diff
@@ -18,12 +18,20 @@ let package = Package(
         .executable(name: "APIGateway", targets: ["APIGateway"]),
         // fully featured example with domain specific business logic
         .executable(name: "CurrencyExchange", targets: ["CurrencyExchange"]),
```

Comment:
@tomerd / @fabianfett we need to remove _all_ executables from the `products`. If you export it as a `product`, then other libraries can depend on them. `swift run` works on all `targets`, so just leave out everything that's not meant for other packages to depend on.

Reply:
Thanks @fabianfett for pointing out to me that this isn't the main Package.swift. So we don't need to remove those here. They still aren't necessary but not an issue to have them.